.github/workflows/checks.yml

```diff
@@ -0,0 +1,39 @@
+name: "PR Checks"
+
+on: [pull_request]
+
+jobs:
+  check_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.6
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Restore yarn cache
+        uses: actions/cache@v4
```

Comment:
nit: we started pinning actions by commit sha instead of tag for security reasons

Reply:
https://github.com/ExodusMovement/fork-sync/pull/4/commits/edc12fa4bc5cc1747d58f7195320258ea961a5a9